installers-conda/build_installers.py

```diff
@@ -155,7 +155,7 @@
     if k == "spyder":
         SPYVER = v[-1]
 
-OUTPUT_FILE = DIST / f"{APP}-{SPYVER}-{OS}-{ARCH}.{args.install_type}"
+OUTPUT_FILE = DIST / f"{APP}-{OS}-{ARCH}.{args.install_type}"
```

Comment:
A couple of questions here:

* Is `OS` really necessary? Given that users will probably get the installers from our webpage or a link in the Releases page, and file extensions will let them identify the OS easily, I think that's not really necessary.
* Should this be `{APP}_{ARCH}` to follow a similar naming scheme as in Spyder 5?

Reply:
I don't feel strongly about `-` vs `_`. I'm happy to use the latter.

While the extension is unique, I feel that the extension is not as obvious to most people and the explicit OS is more readily identifiable. `constructor` can also make a `.sh` installer for macOS; though we aren't using it, it may not be obvious to users that our `.sh` installer _cannot_ be used on macOS. I won't push back too much here, though.

@CAM-Gerlach @dalthviz, what do you think?

Reply:
Sorry for the delay; at first I was going to agree with @ccordoba12 to avoid the website and docs download links breaking again and having to update them. However, I realize now that in fact most if not all of the links will break anyway, because the 5.x installer names themselves are highly inconsistent between platforms in these respects and more (arch naming, install type naming, `_` vs `-`, capitalization, etc)—which, to note, was a concern I raised when we first introduced the installers and the corresponding links to them, but no action was taken :wink: 

So, therefore, since we're going to need to update our existing links anyway, we may as well just make them descriptive and maximally consistent (which would also allow us to generate them programmatically in the future rather than hardcoding), as @mrclary has them above: including OS for clarity, and using `-` instead of `_` as first-level separators, per standard convention (as `_` is already used as a second-level separator within individual components, and is considered internal to "words" rather than separating them by text selection, regex, editors, etc).

Reply:
Okay, so @CAM-Gerlach votes to keep OS and dashes. I tend to agree. @ccordoba12 @dalthviz any response or rebuttal?

Reply:
> which, to note, was a concern I raised when we first introduced the installers and the corresponding links to them, but no action was taken 😉

I'm sorry I overlooked this @CAM-Gerlach. But you are correct, we should have addressed this much earlier.

Reply:
Hi there, I don't have any strong opinion over `-` vs `_` but being explicit about OS sounds good to me :+1:

Reply:
I don't have a strong opinion about using OS, so it's fine for me to add it to the new installers. And I agree with @CAM-Gerlach about using `-` first and then `_`.